nornir/plugins/tasks/version_control/gitlab.py

```diff
@@ -36,16 +36,16 @@ def _get_repository(session: requests.Session, url: str, repository: str) -> int
     for p in respjson:
         if p.get("name", "") == repository:
             found = True
-            pid = p.get("id", 0)
+            pid = p.get("id", "0")
```

Comment:
Why are you converting the default value to a str here? This doesn't make sense to me.

Reply:
Its make sense for type annotations...